ovs/extensions/hypervisor/apis/kvm/sdk.py

```diff
@@ -387,6 +387,44 @@ def clone_vm(self, vmid, name, disks, mountpoint):
         return self.create_vm_from_template(name, source_vm, disks, mountpoint)
 
 
+    def create_volume(self, location, size):
+        """
+        Create volume using truncate
+        """
+        if self.ssh_client.file_exists(location):
+            raise RuntimeError('File already exists at %s' % location)
+        command = 'truncate -s {0}G "{1}"'.format(size, location)
+        output = self.ssh_client.run(command)
+        if not self.ssh_client.file_exists(location):
+            raise RuntimeError('Cannot create file %s. Output: %s' % (location, output))
+        logger.info('Command {0}. Output {1}'.format(command, output))
+
+    def delete_volume(self, location):
+        """
+        Remove volume using rm
+        """
+        if not self.ssh_client.file_exists(location):
+            logger.error('File already deleted at %s' % location)
+            return
+        command = 'rm "{0}"'.format(location)
+        output = self.ssh_client.run(command)
+        logger.info('Command {0}. Output {1}'.format(command, output))
+        if self.ssh_client.file_exists(location):
+            raise RuntimeError('Could not delete file %s, check logs. Output: %s' % (location, output))
+        if output == '':
```

Comment:
I think the run will always raise if the rc is not none. But otoh, a few lines above, it is validated that the file actually is gone, so does the output matter anyway here?